pkgs/racket-doc/scribblings/reference/serialization.scrbl

```diff
@@ -583,6 +583,38 @@ This directory path is used as a last resort when
 relative path with respect to the top level. Usually, it should be
 @racket[(or (current-load-relative-directory) (current-directory))].}
 
+@examples[
+ #:eval ser-eval
+ (struct pie (type)
+   #:mutable
+   #:property prop:serializable
+   (make-serialize-info
+    (λ (this)
+      (vector (pie-type this)))
+    'pie-beam
+    #t
+    (or (current-load-relative-directory) (current-directory))))
+ (define pie-beam
+   (make-deserialize-info
+    (λ (type)
+      (pie type))
+    (λ ()
+      (define pie-pattern (pie 'transporter-error))
+      (values pie-pattern
+              (λ (type)
+                (set-pie-type! pie-pattern type))))))
+ (define original-pie
+   (pie 'apple))
+ original-pie
+ (define pie-in-transit
+   (serialize original-pie))
+ pie-in-transit
```

Comment:
Does this (and the `(pie-type ....)` call below) look good when rendered?

(For me, "good" is "takes up less than 1 line".)

Reply:
I think so. At least this is how it looks like when I render it.
![screen shot 2017-09-26 at 4 13 02 pm](https://user-images.githubusercontent.com/285926/30882015-b70f0c26-a2d5-11e7-8e65-a85187e3d1da.png)